configure.ac

```diff
@@ -179,7 +179,10 @@ AC_CHECK_TYPES([struct file_handle],,,[
 #include <fcntl.h>
 ])
 
-AC_CHECK_TYPES([struct fs_quota_statv],,,[#include <xfs/xqm.h>])
+AC_CHECK_TYPES([struct fs_quota_statv],,,[
+#define _GNU_SOURCE
```

Comment:
What exactly you are trying to fix?

`struct fs_quota_statv` is not guarded by `_GNU_SOURCE`: https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/tree/include/xqm.h. Therefore my system check for `struct fs_quota_statv` works as is (without `_GNU_SOURCE`).

Quoting your commit message:
> Required as xfs/xqm.h uses fallocate() which is only defined when _GNU_SOURCE is defined.`

None of our quotactl tests use `fallocate()`.

I see only musl defines `fallocate64` when `_GNU_SOURCE` defined, but I don't see how this could be used here.

```
#if defined(_GNU_SOURCE)
#define fallocate64 fallocate
#endif
````